### ISSUE TITLE
[Feature Request]: Bring over the `KryptonCommandLinkButtons`

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#1187]
Is it time to bring over the `KryptonCommandLinkButtons` from the Extended Toolkit?

**Comment by Wagnerp** [comment]
> Please re-open this, so that the "Class" is actually called `KryptonCommandLinkButton` please ?

@Smurf-IV There were 2 types, what should the second one be named as? Currently it's `KryptonAlternateCommandLinkButton` & the second one is `KryptonCommandLinkButton`.

**Comment by Smurf-IV** [comment]
I thought only one of them was actually worth using in the Ext kit.
Does the demo use both ?

**Comment by Wagnerp** [comment]
> I thought only one of them was actually worth using in the Ext kit. Does the demo use both ?

@Smurf-IV Yes, the one which looks like a normal KButton is the `KryptonAlternateCommandLinkButton` and vice-versa. I can switch the names around?

**Comment by Smurf-IV** [comment]
I think it should replace the one that does not do stuff in the demo.
i.e. the one that works should have the expected name.

**Comment by Wagnerp** [comment]
> I think it should replace the one that does not do stuff in the demo. i.e. the one that works should have the expected name.

I've 'hidden' the `KryptonAlternateCommandLinkButton` version, so it can be revisited at a later date if needed.
